run_model.py

```diff
@@ -61,6 +70,12 @@ def run_state(state=None, deploy=False):
     run.run_state_level_forecast(
         min_date, max_date, country="USA", state=state, output_dir=output_dir
     )
+    logging.info(f'Wrote output to {output_dir}')
+    # only write the version if we saved everything
+    if state is None :
+        version.write_file('states', output_dir)
```

Comment:
Maybe in the future we can write this in the json file itself

Reply:
I like this idea, but we would obviously need to verify that the consumers don't break.